tests/test_parse_requirements.py

```diff
@@ -45,18 +45,33 @@ def test_parse_requirements_with_index_url(monkeypatch, flag):
     "https://github.com/pypa/pip/archive/1.3.1.zip"
     "#sha1=da9234ee9982d4bbb3c72346a6de940a148ea686"
 )
+PEP508_PIP_EXAMPLE_WHEEL = (
+    "https://github.com/pypa/pip/archive/pip-1.3.1-py2.py3-none-any.whl"
+)
+PEP508_PIP_EXAMPLE_EGG = (
+    "ssh://git@github.com/pypa/pip.git@da9234ee9982d4bbb3c72346a6de940a148ea686#egg=pip"
+)
+PEP508_PIP_EXAMPLE_EGG_FILE = "file://tmp/pip-1.3.1.zip#egg=pip"
+PEP508_PIP_EXAMPLE_EGG_WHEEL = "file://tmp/pip-1.3.1-py2.py3-none-any.whl"
```

Comment:
I'm not sure the names of these variables make sense, the first one is just a direct URL with an egg fragment, and the second one is just a regular wheel, what does it have to do with eggs?

Reply:
Yeah my bad, that's supposed to say `PEP508_PIP_EXAMPLE_WHEEL_FILE`.

Reply:
d332fb9